ci/upload-to-concourse.sh

```diff
@@ -2,35 +2,16 @@
 
 set -e
 
-if [ -z "$CONCOURSE_TARGET" ]; then
-    echo "Missing CONCOURSE_TARGET env var."
-    exit 1
-fi
-
-if [ -z "$CONCOURSE_URL" ]; then
-    echo "Missing CONCOURSE_URL env var."
-    exit 1
-fi
-
-if [ -z "$CONCOURSE_USER" ]; then
-    echo "Missing CONCOURSE_USER env var."
-    exit 1
-fi
-
-if [ -z "$CONCOURSE_PASSWORD" ]; then
-    echo "Missing CONCOURSE_PASSWORD env var."
-    exit 1
-fi
-
 if [ ! -f "vars.yml" ]; then
     echo "Missing vars.yml. Please create it first."
     exit 1
 fi
 
+echo "Please follow https://github.com/cloudfoundry/routing-concourse/blob/main/README.md#authentication-to-concourse for login."
 
-fly -t "$CONCOURSE_TARGET" login -c "$CONCOURSE_URL" -u "$CONCOURSE_USER" -p "$CONCOURSE_PASSWORD"
-fly -t "$CONCOURSE_TARGET" validate-pipeline -c pipeline.yml
-fly -t "$CONCOURSE_TARGET" set-pipeline -p haproxy-boshrelease -c pipeline.yml --load-vars-from vars.yml
-fly -t "$CONCOURSE_TARGET" expose-pipeline -p haproxy-boshrelease
+fly -t cfi login -c https://concourse.arp.cloudfoundry.org/
```

Comment:
Would it still be possible to change the team name? CFI is an outdated reference to an SAP internal team name, something like `networking-extensions` would be more appropriate.

Reply:
It can be any name, I have re-used what we mention in other docs: https://pages.github.tools.sap/cloudfoundry/documentation/internal-documentation/cfn-haproxy-boshrelease-os#hijack-latest-concourse-jobs
I use `concourse-arp` internally. It can be anything, but needs updating in both places (here and any other docs).

Reply:
Could you then please rename it to `networking-extensions` as the pipelines are from that WG area? It will also allow us to separate the other pipelines by area to give proper permissions to reviewers / approvers of the respective area.

<sub>And please try to keep references to SAP internal resources out of GitHub.com</sub>

Reply:
or `arp`?